File: Web/Vaishnavi-Singh20002_VaishnaviSingh_2125ec1060_02/Patterns/pattern11.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
   int number=1;
   int s=1;
   for(int i=1;i<=n;i++){
       for(int j=1;j<=s;j++){
           cout<<number<< "\t" ;
           number++;
       }
       cout<<"\n" ;
       s++;
   }
    
    
    //write your code here
    
}

File: Android/sukritoberoi_SukritKaurOberoi_2125cse1065_2/Patterns/Pattern11.java
import java.util.*;

public class Main {

  public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int n,i,j,c=1;
    n=scn.nextInt();
    for(i=1;i<=n;i++)
    {
        for(j=1;j<=i;j++)
        {
            System.out.print(c+"\t");
            c++;
        }
        System.out.println();
    }


    // write ur code here

  }
}